style.css

```diff
@@ -130,11 +131,14 @@ u {
 button,
 input[type="submit"],
 input[type="reset"] {
+  all: unset;
+  color: var(--button-text);
   box-sizing: border-box;
   border: none;
   background: var(--surface);
   box-shadow: var(--border-raised-outer), var(--border-raised-inner);
-  border-radius: 0;
+  border-radius: 0px;
```

Comment:
nit to use `0` over `0px`. not currently linting for this

Reply:
solved :))